src/components/mypage/ReviewModal/index.tsx

```diff
@@ -0,0 +1,156 @@
+  }
+
+  const handleInput: ChangeEventHandler<HTMLTextAreaElement> = e => {
+    setReviewState({
+      ...reviewState,
+      reviewText: e.target.value
+    })
+  }
+
+  return (
+    <Styled.ReviewModal isOpen={isOpen} onClose={onClose}>
+      <CommonTitleContainer
+        nickname={nickname}
+        productName={productName}></CommonTitleContainer>
+      <Styled.ReviewIconContainer>
+        {MOCK_SCORE.map(scoreItem => {
+          return (
+            <Styled.ReviewState
+              key={scoreItem.state}
+              onClick={(): void => handleClickReviewIcon(scoreItem.state)}>
+              <Styled.ReviewIcon
+                isFill={reviewState.reviewScore === scoreItem.state}
+                isGood={scoreItem.state === 'smile'}
+                type={
+                  reviewState.reviewScore === scoreItem.state
+                    ? `${scoreItem.state}Fill`
+                    : scoreItem.state
+                }></Styled.ReviewIcon>
+              <Styled.ReviewText
+                isFill={reviewState.reviewScore === scoreItem.state}>
+                {scoreItem.text}
+              </Styled.ReviewText>
+            </Styled.ReviewState>
+          )
+        })}
+      </Styled.ReviewIconContainer>
+
+      <Styled.ReviewTextArea
+        guideMessage={`${reviewState.reviewText?.length}/100`}
+        maxLength={100}
+        onInput={handleInput}></Styled.ReviewTextArea>
+
+      <Styled.ReviewSendButton
+        disabled={!isClickReviewIcon}
+        styleType={isClickReviewIcon ? 'solidPrimary' : 'solidDisabled'}
+        onClick={(): void => onConfirm(reviewState)}>
+        {'후기 보내기'}
+      </Styled.ReviewSendButton>
+    </Styled.ReviewModal>
+  )
+}
+
+const ReadReviewModal = ({
+  isOpen = true,
+  onClose,
+  onConfirm,
+  nickname = '닉네임',
+  productName = '상품이름',
+  score = 'smile',
+  content = '리뷰'
+}: ReadReviewModalProps): ReactElement => {
+  return (
+    <Styled.ReviewModal isOpen={isOpen} onClose={onClose}>
+      <Styled.TitleContainer>
+        <Styled.FirstSection>
+          <Styled.NickName>{nickname}</Styled.NickName>
+          <Styled.NormalText>님과</Styled.NormalText>
+        </Styled.FirstSection>
+        <Styled.NormalText>거래는 어땠나요?</Styled.NormalText>
+        <Styled.ProductText>{productName}</Styled.ProductText>
+      </Styled.TitleContainer>
+      <Styled.ReviewIconContainer>
+        <Styled.ReviewState>
+          <Styled.ReviewIcon
+            isFill
+            isGood={score === 'smile'}
+            type={`${score}Fill`}></Styled.ReviewIcon>
+          <Styled.ReviewText isFill>
+            {MOCK_SCORE.find(scoreItem => scoreItem.state === score)?.text}
+          </Styled.ReviewText>
+        </Styled.ReviewState>
+      </Styled.ReviewIconContainer>
+      <Styled.ReadModeReviewContent>{content}</Styled.ReadModeReviewContent>
+      <Styled.ReviewSendButton
+        styleType={'solidPrimary'}
+        onClick={(): void => onConfirm()}>
```

Comment:
```suggestion
        onClick={onConfirm}>
```
여기도 바로 적어주어도 될 것 같습니다!

Reply:
완완~
[링크](https://github.com/price-offer/offer-fe/pull/108/commits/fee8074b8f9f054f36734eeda9c8b6bd81a57cc9#diff-e09a102d293983acddee81c811bd4deba04f565e5018032ce85f3f73c603f943)